XenAdmin/Dialogs/OptionsPages/ConnectionOptionsPage.cs

```diff
@@ -92,6 +87,19 @@ private void build()
             if (Registry.ProxyAuthenticationEnabled)
             {
                 AuthenticationCheckBox.Checked = Properties.Settings.Default.ProvideProxyAuthentication;
+                
+                switch ((HTTPHelper.ProxyAuthenticationMethod)Properties.Settings.Default.ProxyAuthenticationMethod)
+                {
+                    case HTTPHelper.ProxyAuthenticationMethod.Basic:
+                        BasicRadioButton.Checked = true;
+                        break;
+                    case HTTPHelper.ProxyAuthenticationMethod.Digest:

```

Comment:
"case HTTPHelper.ProxyAuthenticationMethod.Digest" is not really needed here, as we do the same thing for default case. Actually, I think it would be enough to do just:
`BasicRadioButton.Checked = (HTTPHelper.ProxyAuthenticationMethod)Properties.Settings.Default.ProxyAuthenticationMethod == HTTPHelper.ProxyAuthenticationMethod.Basic;`
because the other radio button will be checked by default if this is unchecked.